packages/flutter_tools/test/general.shard/plugins_test.dart

```diff
@@ -210,7 +211,7 @@ void main() {
             ? fakePubCache.childDirectory(name)
             : fileSystem.directory(nameOrPath);
         packagesFile.writeAsStringSync(
-            '$name:file://${pluginDirectory.childFile('lib').uri}\n',
+            '$name:${pluginDirectory.childFile('lib').uri}\n',
```

Comment:
this was a pre-existing bug that apparently didn't affect any of the existing test results (but did affect my new test). calling `pluginDirectory.childFile('lib').uri` already has the `file://` prefix, so we were getting strings with `file://file://`